crates/uv-resolver/src/lock.rs

```diff
@@ -52,6 +52,11 @@ impl Lock {
         Lock::try_from(wire)
     }
 
+    /// Returns the [`Distribution`] entries in this lock.
+    pub fn distributions(&self) -> &[Distribution] {
```

Comment:
@BurntSushi - Let me know if you have a preferred way to do this (other than what's here). I expose `Distribution`, but none of the fields or methods are `pub`.